### PR TITLE
New API contracts added for routes /open and /stale.

### PR DESCRIPTION
**Pull request by adnanraghib1308**
New API contracts added for routes /open and /stale. Plus modified the contract for pull-requests as its route was changed form /pullrequests/:id to /pullrequests/user/:id.

### CODE REVIEW

**Review by swarajpure — CHANGES_REQUESTED** [review]
Please check `Users` routes to get an idea of how it is supposed to be.
All the pullrequests routes should go in one file only.

Just shifting things here and there. Nothing major I guess. Thank you! 😄

**Comment by adnanraghib1308** [comment]
I have made the requested changes. Please review @swarajpure

File: pull-requests/README.md [hunk 1]
```diff
@@ -19,10 +20,12 @@
...
 |               Route                |           Description           |
 | :--------------------------------: | :-----------------------------: |
-|      [GET /pullrequests/:id](#get-pullrequestsid)      | Returns latest PRs by the user in RDS |
+|      [GET /pullrequests/user/:id](#get-pullrequestsusserid)      | Returns latest PRs by the user in RDS |
+|      [GET /pullrequests/open](#get-pullrequestsopen)      | Returns 10 latest open PRs in RDS |
+|      [GET /pullrequests/stale](#get-pullrequestsstale)       | Returns stale PRs in RDS |
 
 
-## **GET /pullrequests/:id**
+## **GET /pullrequests/user/:id**
```
Comment (swarajpure) [inline pull-requests/README.md:28]: One small change please: we are not passing `id` anymore. We are passing `username` now.
Can you please change it at the places it is needed? 

Also on line 23, when I click on the title, it's not taking me to the required id. You can verify that using either online Markdown editors or extension for VSCode or in Github preview as well. 

I know I was supposed to do it 😅
Let me know if that's feasible or I'll do it.

**Review by swarajpure — COMMENTED** [review]
Just a small change 😅 
Can be merged after that 👍🏻

File: pull-requests/README.md (hunk 1 above)
Comment (adnanraghib1308) [reply pull-requests/README.md]: No problem @swarajpure . Its done

**Review by swarajpure — APPROVED** [review]
Thank you! Great job! 🥳 🎉

**Pull request merged**